tests/test_nohup_script.py

```
@@ -27,23 +27,23 @@ def outputs_dir(self, script_dir):
     
     def test_script_exists_after_rename(self, script_dir):
         """Test that run_nohup.sh exists after renaming."""
-        script_path = script_dir / "run_nohup.sh"
-        assert script_path.exists(), "run_nohup.sh should exist"
+        script_path = script_dir / "scripts" / "run_nohup.sh"
+        assert script_path.exists(), "scripts/run_nohup.sh should exist"
         assert os.access(script_path, os.X_OK), "run_nohup.sh should be executable"
     
     def test_outputs_directory_used(self, script_dir, outputs_dir):
```

Ruff (0.14.5)

Unused method argument: `outputs_dir`

(ARG002)